scipioninstaller/installer.py

```diff
@@ -231,6 +242,10 @@ def main():
                                  'make sure to write the correct path where '
                                  'Scipion will be installed',
                             action='store_true')
+        parser.add_argument('-scipionEnv', help='Name of the virtual environment. '
+                                             'By default, if this parameter is '
+                                             'not passed, the name will be '
+                                             '.scipion3env')
```

Comment:
Isn't there a parameter for a default value? If so use it (with the SCIPION_ENV please) and lines below are not needed.